go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
 	github.com/imdario/mergo v0.3.7
 	github.com/kubeless/kubeless v1.0.7
+	github.com/mitchellh/gox v1.0.1 // indirect
```

Comment:
This package is only used in the CI pipeline. Please revert the changes in `go.mod` and `go.sum`.

Reply:
changes reverted